template/packages/frontend/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["next/babel"],
+  "plugins": [["styled-components", { "ssr": true }]]
+}
```

Comment:
The styled-components babel plugin does a couple of things for us. In SSR mode it sets up consistent hashing for generated class names, and does some logic to make sure that generated class names match up between server-side and client rendering. That's important to make rehydration on the client work smoothly. It also inserts component names into generated class names to make it easier to see which component each set of style rules came from.